<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsChoicesWindow</class>
 <widget class="QMainWindow" name="clsChoicesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choices</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="8">
     <widget class="QLabel" name="lblQuestion">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;What would you like to do?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="5">
     <widget class="QPushButton" name="btnBuildHorizontal">
      <property name="text">
       <string>Build horizontally</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QPushButton" name="btnNerdPole">
      <property name="text">
       <string>Nerdpole</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QPushButton" name="btnMine">
      <property name="text">
       <string>Mine</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QPushButton" name="btnFish">
      <property name="text">
       <string>Fish</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="5">
     <widget class="QLabel" name="lblHoldBtn">
      <property name="text">
       <string>Hold button:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="5">
     <widget class="QLabel" name="lblRapidFire">
      <property name="text">
       <string>Rapid fire button:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="3">
     <layout class="QHBoxLayout" name="btnLeftRightHold">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="btnLeftHold">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRightHold">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inpCustomHold">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="5" colspan="3">
     <layout class="QHBoxLayout" name="btnLeftRightRapidFire">
      <item>
       <widget class="QPushButton" name="btnRapidFireLeft">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRapidFireRight">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inpCustomRapidFire">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actAboutQT"/>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actAboutQT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
